huaweicloud/services/acceptance/mrs/resource_huaweicloud_mapreduce_cluster_test.go

```diff
@@ -1550,3 +1550,119 @@ resource "huaweicloud_mapreduce_cluster" "test" {
 }
 `, testAccMrsMapReduceClusterConfig_base(rName), rName, pwd, acceptance.HW_MAPREDUCE_BOOTSTRAP_SCRIPT)
 }
+
+func TestAccMrsMapReduceCluster_alarm(t *testing.T) {
+	var clusterGet cluster.Cluster
+	resourceName := "huaweicloud_mapreduce_cluster.test"
+	rName := acceptance.RandomAccResourceNameWithDash()
+	password := acceptance.RandomPassword()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckMRSV2ClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMrsMapReduceClusterConfig_alarm(rName, password),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMRSV2ClusterExists(resourceName, &clusterGet),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "type", "CUSTOM"),
+					resource.TestCheckResourceAttr(resourceName, "safe_mode", "false"),
+					resource.TestCheckResourceAttr(resourceName, "status", "running"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"manager_admin_pass",
+					"node_admin_pass",
+					"template_id",
+					"smn_notify",
```

Comment:
if we add `smn_notify` in the ignoring list, we should add it in **Import** section in docs.

Reply:
fixed